src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
+    
```

Comment:
Tooling-related changes presumably? Do we want to keep them?

Reply:
Presumably VS changed it, I will revert since it looks like it adds unnecessary spaces

Reply:
will be reverted in the next iteration